docs/manual/index.md

```diff
@@ -14,4 +14,5 @@ This document assumes that you are generally familiar with Kubernetes. For more
 1. [Controlling Fault Domains](fault_domains.md)
 1. [Running with TLS](tls.md)
 1. [Backup](backup.md)
+1. [Coordinator Selection](coordinator_selection.md)
```

Comment:
If we're going to introduce this as a new document, it would be good to have a section at the bottom linking to the next document, and to update the `Backup` document to link to `Coordinator Selection`. On the other hand, it's short enough that it might be fine to fit it into the `Controlling Fault Domains` doc, which already talks about disruption budgets and talks about how coordinators are selected. We'll need to update that doc one way or the other.

Reply:
I think merging it into `Controlling Fault Domains` make sense.